src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -288,6 +290,7 @@ class SaaSConfig(SaaSConfigBase):
     endpoints: List[Endpoint]
     test_request: SaaSRequest
     data_protection_request: Optional[SaaSRequest] = None  # GDPR Delete
+    rate_limits: Optional[List[RateLimitConfig]]
```

Comment:
At the moment I would just like to put `rate_limits` in the `SaaSRequest` and `SaaSConfig`. I found that trying to add it to the `ConnectionConfig` object would have added alot of complexity. Happy to discuss this further if we want to. Also, since we are not integrating the rate limiter with all connectors(yet) it didn't seem worth it to go this route.

Reply:
On first pass I couldn't find a simple way to validate the object that was being inserted into the database. To add this field to `ConnectionConfig` it would have needed to have fit a type which could be passed into `Column`.  It looks like it might be possible to add a `JSONB` column and use `@validates`. 

Will have do some testing with this one.

Reply:
OK, thanks for explaining that. seems fine to punt on it for now given the additional complexity.